templates/options.conf.erb

```diff
@@ -4,3 +4,6 @@ forwarders { <%= scope.lookupvar('::dns::forwarders').join("; ") %>; };
 <% end -%>
 
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
+<% if (@operatingsystem == 'FreeBSD') -%>
```

Comment:
He doesn't want to change the package for various reasons, so this would need to stay in (or handled via puppet and set if some variable is not undef)... your opinion?